MapTheVoteAddressBuilder/MapTheVoteAddressBuilder.cs

```diff
@@ -39,13 +39,11 @@ static void Main(string[] args)
             Console.WriteLine("By: CJ Stankovich https://github.com/siegeJ");
             Console.ForegroundColor = ConsoleColor.White;
 
+            string JSESSIONID = "";
```

Comment:
I think you had it right the first time since below it uses tr.JESSIONID

Reply:
Actually looks like we can remove this tr object all together since we aren't using these coordinates for anything anymore.

Reply:
Yeah, good point. The debug coords are set from another place now. I'll get this removed.